tests/unit/test_models.py

```diff
@@ -1061,3 +1062,71 @@ def test_coderepo_from_workspace(bll):
     CodeRepo.from_workspace(
         workspace, workspace.manifest["outputs"]["foo.txt"]["commit"]
     )
+
+
+def test_displayed_decision_for_conflicting_file_reviews(bll):
+    path = UrlPath("path/file1.txt")
+
+    author = factories.create_airlock_user(
+        username="author", workspaces=["workspace"], output_checker=True
+    )
+
```

Comment:
It'd be worth adding another user here, who has access to the workspace, but isn't an output checker, and checking that they also see changes_requested for a conflicted file (after return)